modules/javafx.graphics/src/main/docs/javafx/scene/doc-files/cssref.html

```diff
@@ -2002,41 +2000,63 @@ <h4><a id="node">Node</a></h4>
+          <th class="subheader" scope="col">Comments</th>
         </tr>
         <tr>
-        <th class="propertyname" scope="row">focused</th>
+          <th class="propertyname" scope="row">focused</th>
           <td>applies when the <strong>focused</strong> variable is true</td>
         </tr>
         <tr>
-        <th class="propertyname" scope="row">focus-visible</th>
+          <th class="propertyname" scope="row">focus-visible</th>
           <td>applies when the <strong>focusVisible</strong> variable is true</td>
         </tr>
         <tr>
-        <th class="propertyname" scope="row">focus-within</th>
+          <th class="propertyname" scope="row">focus-within</th>
           <td>applies when the <strong>focusWithin</strong> variable is true</td>
         </tr>
         <tr>
-        <th class="propertyname" scope="row">hover</th>
+          <th class="propertyname" scope="row">hover</th>
           <td>applies when the <strong>hover</strong> variable is true</td>
         </tr>
         <tr>
-        <th class="propertyname" scope="row">pressed</th>
+          <th class="propertyname" scope="row">pressed</th>
           <td>applies when the <strong>pressed</strong> variable is true</td>
         </tr>
         <tr>
-        <th class="propertyname" scope="row">show-mnemonic</th>
-          <td>apples when the mnemonic affordance (typically an underscore)
-            should be shown.</td>
+          <th class="propertyname subheader" scope="col">Input Pseudo-classes</th>
+          <th class="subheader" scope="col">Comments</th>
+        </tr>
+        <tr>
+          <th class="propertyname" scope="row">disabled</th>
+          <td>applies when the <strong>disabled</strong> variable is true</td>
+        </tr>
+        <tr>
+          <th class="propertyname" scope="row">show-mnemonic</th>
+          <td>applies when the mnemonic affordance (typically an underscore) should be shown</td>
+        </tr>
+        <tr>
+          <th class="propertyname subheader" scope="col">Tree-Structural Pseudo-classes</th>
+          <th class="subheader" scope="col">Comments</th>
+        </tr>
+        <tr>
+          <th class="propertyname" scope="row">first-child</th>
+          <td>applies when the node is the first child in its <code>Parent</code> container</td>
+        </tr>
+        <tr>
+          <th class="propertyname" scope="row">last-child</th>
+          <td>applies when the node is the last child in its <code>Parent</code> container</td>
+        </tr>
+        <tr>
+          <th class="propertyname" scope="row">only-child</th>
+          <td>applies when the node is the only child in its <code>Parent</code> container</td>
+        </tr>
+        <tr>
+          <th class="propertyname" scope="row">nth-child()</th>
+          <td>applies when the node is the n-th child in its <code>Parent</code> container
+              (the only acceptable arguments are "even" and "odd")</td>
```

Comment:
should we also specify that the numbering is 1-based?
i.e. the child at index 0 is the first (1st) and therefore `ODD`?

Reply:
While I was reviewing, I had to look that up so I could check if the code was doing the correct thing. CSS indeed considers the first child as having index 1, even though it will be index 0 in the list returned by `getChildren`.  It may be worth pointing that out explicitly.